src/Analyzers/VisualBasic/CodeFixes/RemoveUnusedParametersAndValues/VisualBasicRemoveUnusedValuesCodeFixProvider.vb

```diff
@@ -63,6 +63,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.RemoveUnusedParametersAndValues
             Throw New NotImplementedException()
         End Function
 
+        Protected Overrides Function GetReplacementNodeForVarPattern(originalVarPattern As SyntaxNode, newNameNode As SyntaxNode) As SyntaxNode
+            ' VB does not have var patterns
+            Throw New NotImplementedException()
```

Comment:
```suggestion
            Throw ExceptionUtilities.Unreachable
```

Reply:
Both of them, right?)

Reply:
You can do both, or restrict it to just the method you added.